hspec-core/src/Test/Hspec/Core/Formatters/V2.hs

```diff
@@ -293,7 +294,7 @@ defaultFailedFormatter = do
         Error _ e -> withFailColor . indent $ (("uncaught exception: " ++) . formatException) e
 
       writeLine ""
-      writeLine ("  To rerun use: --match " ++ show (joinPath path))
+      writeLine ("  To rerun use: --match " ++ ushow (joinPath path))
```

Comment:
From what I understand, this is the only change that makes an actual difference.  @ncaq is that the case?  Can we revert the other changes?

Reply:
I decided to use ushow in the formatting module because I thought that escape sequences are nothing to be happy about appearing as a result of formatting.
If you have a strong desire to keep the show, we can put it back.

Reply:
Yes, please, let's keep `show` except for when `ushow` is needed.